adoc/extensions/sycl_khr_graph.adoc

```diff
@@ -1,14 +1,29 @@
-= sycl_khr_command_graph
+= sycl_khr_graph
+
+== Overview
+
+The graph abstraction defines an explicitly lazy execution model by decoupling
+command creation from submission.
+These concepts are potentially tied together in SYCL implementations that
+eagerly submit work to a device when a command-group is submitted to a queue.
```

Comment:
Could link to the command group section of the spec here with `<<command-group>>`. I'm not exactly sure how it'll render that link so `<<command-group, command-group>>` might be safer to definitely keep the same wording.